components/datetime/src/format/mod.rs

```diff
@@ -1,11 +1,15 @@
 // This file is part of ICU4X. For terms of use, please see the file
```

Comment:
Why did you turn `format.rs` into `format/mod.rs`?

Reply:
Oops, this was leftover from before we were using `StructProvider` to let these tests be integration tests. It looks like I missed this change when I was converting everything back. I will return it to how it was.